packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_RuntimeSerializationHelper.ts

```diff
@@ -108,6 +108,8 @@ export const V1_serializeRuntime = (
     return serialize(V1_EngineRuntime, protocol);
   } else if (protocol instanceof V1_RuntimePointer) {
     return serialize(V1_runtimePointerModelSchema, protocol);
+  } else if (protocol instanceof V1_LegacyRuntime) {
```

Comment:
Sorry, why do we need this? We intentionally do this so that `legacy runtime` will never be in circulation again. I don't get why we would need this anyhow.